content/docs/deploy/enterprise/changelog.mdx

```diff
@@ -7,6 +7,18 @@ toc_max_heading_level: 5
 
 # Changelog
 
+## 0.23.0
+
+### New
+
+- [Set Request Headers](/docs/reference/routes/headers#set-request-headers) has a new token substitution value that sends the client certificate fingerprint (downstream mTLS must be enabled) to the upstream application or service.
```

Comment:
We should probably mention the ID token and access token substitutions here as well.

Reply:
It looks like we started co-mingling upgrade notes and new functionality around v0.17. Traditionally, upgrade guide should _just_ be what the user needs to know before they upgrade, and any steps they need to take. I think we should get back to that and leave text like this in the announcement. We can link to the announcement posts here however which might be nice.